### PR TITLE
Generate v0.0.2 Bundle

### PR DESCRIPTION
**Pull request by anishasthana**
This PR introduces a make target to sync CRDS, in addition to syncing the CRDS.

### CODE REVIEW

File: bundle/manifests/codeflare-operator.clusterserviceversion.yaml
Comment (tedhtchang) [inline bundle/manifests/codeflare-operator.clusterserviceversion.yaml:1]: The CSV version still 0.0.1. Did you run `VERSION=0.0.2 make bundle`

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -5,6 +5,12 @@ generatorOptions:
...
-  files:
+- files:
   - controller_manager_config.yaml
+  name: manager-config
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: quay.io/project-codeflare/codeflare-operator
+  newTag: v0.0.1
```
Comment (tedhtchang) [inline config/manager/kustomization.yaml:16]: v0.0.2

File: config/manifests/kustomization.yaml [hunk 2]
```diff
@@ -5,23 +5,9 @@ resources:
...
-#    - op: remove
-#      path: /spec/template/spec/containers/1/volumeMounts/0
-#    # Remove the "cert" volume, since OLM will create and mount a set of certs.
-#    # Update the indices in this path if adding or removing volumes in the manager's Deployment.
-#    - op: remove
-#      path: /spec/template/spec/volumes/0
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+patches:
+- patch: '[{"op":"add", "path":"/metadata/annotations/containerImage", "value": "quay.io/project-codeflare/codeflare-operator:v0.0.1" }]'
```
Comment (tedhtchang) [inline config/manifests/kustomization.yaml:11]: v0.0.2

**Review by tedhtchang — CHANGES_REQUESTED** [review]
The versions are still 0.0.1 in a few places.

**Review by jbusche — APPROVED** [review]
LGTM, Tested on my OC 4.12.7 cluster using operator-sdk

File: config/manifests/kustomization.yaml [hunk 3]
```diff
@@ -5,23 +5,12 @@ resources:
...
+  target:
+    kind: ClusterServiceVersion
```
Comment (tedhtchang) [inline config/manifests/kustomization.yaml:13]: Hey @anishasthana Does this seem redundant ?

**Review by tedhtchang — APPROVED** [review]
/approve

File: config/manifests/kustomization.yaml (hunk 3 above)
Comment (tedhtchang) [reply config/manifests/kustomization.yaml:13]: Doesn't seems to cause any problem

**Pull request merged**